03_analysis/compare_ffs.py

```diff
@@ -766,25 +898,56 @@ def main(in_dict, read_pickle, conf_id_tag, plot=False, mol_slice=None):
                 ml,
                 "RMSD ($\mathrm{\AA}$)",
                 "ddE (kcal/mol)",
-                f"density_rmsd_{ml}.png",
+                f"fig_density_rmsd_linear_{ml}.png",
+                "paper",
+                x_range=(0, 3.7),
+                y_range=(-30, 30),
+                z_range=(-260, 5200),
+                z_interp=True,
+                symlog=False)
+
+            draw_density2d(
+                tfds[i],
+                energies[i],
+                ml,
+                "TFD",
+                "ddE (kcal/mol)",
+                f"fig_density_tfd_linear_{ml}.png",
+                "paper",
+                x_range=(0, .8),
+                y_range=(-30, 30),
+                z_range=(-302, 7060),
+                z_interp=True,
+                symlog=False)
+
+            draw_density2d(
+                rmsds[i],
+                energies[i],
+                ml,
+                "RMSD ($\mathrm{\AA}$)",
+                "ddE (kcal/mol)",
+                f"fig_density_rmsd_{ml}.png",
```

Comment:
Maybe use figname of `f"fig_density_rmsd_log_{ml}.png"` here so that all the log figures are organized together

Reply:
Resolved in f0d5033